bubble-wrap.yaml

```diff
@@ -852,7 +852,7 @@ styles:
                 WALL_TINT: vec3(1., 3., .800)
             blocks:
                 color: |
-                    if (dot(vec3(0., 0., 1.), worldNormal()) < 1.0 - TANGRAM_EPSILON) {
+                    if (dot(vec3(0., 0., 1.), worldNormal()) < .5) {
```

Comment:
Interesting.

`0.5` is too sharp of a change though. When I tried this on play on Edge, reducing the `TANGRAM_EPSILON` to `0.001` worked for me.